app/main.py

```diff
@@ -4,8 +4,11 @@
 
 app = FastAPI()
 
-@app.get("/brand/president")
+@app.get("/")
+def hello():
+    return {"message": "Hello from facets-knowledge-panels! Tip: open /docs for documentation"}
```

Comment:
I don't know if it's useful to speak about /docs here, in production no one will see that folder. People will try to add /docs to the url, leading nowhere….

As currently we only have one url, you could just propose to try it :-)
```suggestion
    return {
      "message": "Hello from facets-knowledge-panels! Tip: open /<facet-name>/<facet-value>. Example: /brand/president"
      "docs": "https://github.com/openfoodfacts/facets-knowledge-panels"
    }
```

Reply:
Than let this PR as draft , maybe after some more commit and after having more urls we could merge it.

Reply:
sorry I did not realize, FastAPI was generating a documentation by itself, so your first message was perfectly alright !